frontend/javascripts/oxalis/view/layouting/tracing_layout_view.js

```diff
@@ -183,32 +178,30 @@ class TracingLayoutView extends React.PureComponent<PropsWithRouter, State> {
     });
   };
 
-  handleSettingsCollapse = () => {
-    this.setState(prevState => ({
-      isSettingsCollapsed: !prevState.isSettingsCollapsed,
-    }));
-  };
-
-  onLayoutChange = (layoutConfig, layoutName) => {
+  onLayoutChange = (model?: Object, layoutName?: string) => {
     recalculateInputCatcherSizes();
     window.needsRerender = true;
-    this.currentLayoutConfig = layoutConfig;
-    this.currentLayoutName = layoutName;
+    if (model != null) {
+      this.setState({ model });
+    }
     if (this.props.autoSaveLayouts) {
-      this.saveCurrentLayout();
+      this.saveCurrentLayout(layoutName);
     }
   };
 
-  saveCurrentLayout = () => {
-    if (this.currentLayoutConfig == null || this.currentLayoutName == null) {
-      return;
-    }
+  // eslint-disable-next-line react/sort-comp
+  debouncedOnLayoutChange = _.debounce(
+    () => this.onLayoutChange(),
+    Constants.RESIZE_THROTTLE_TIME / 5,
```

Comment:
The `/ 5` has been there before.
The  `RESIZE_THROTTLE_TIME` variable is also used in the `arbitrary_view`and `plane_view` for throttling update events but without the `/ 5`. It is not clear to me, why one component uses a fifth of the time for throttling and the others the full time :man_shrugging:. 
Maybe we can remove the `/ 5` and reduce `RESIZE_THROTTLE_TIME` by half or two-third :thinking:. At least this would make the code more consistent. But this would mean, that the event in the `arbitrary_view`and `plane_view` are triggered more often.

The events that were throttled with the `RESIZE_THROTTLE_TIME` `arbitrary_view`and `plane_view` are window resizes and `k/l` zooming (which is currently not supported).

But on second thought: Maybe the reduced throttle time in the `tacing_layout_view` is needed because this triggers the on resize event earlier at the resize event in the `arbitrary_view`and `plane_view` because both views depend on the called `onLayoutChange` that updates the input catcher size.

I investigated further: In the current version of this PR, the resize event in `arbitrary_view`and `plane_view` is triggered first and after that comes the resize event in `onLayoutChange` call in `tacing_layout_view`. Therefore I conclude, that the order is not relevant here. Thus we can use a single constant for all cases. As I do not know, why the constant in  `tacing_layout_view` is divided by 5, I simply suggest removing the division :)

If you think that we should also decrease the constant (currently 250) to get a faster response time, let me know :smile_cat:

P.S.: I hope you can understand my not so clear text :sweat_smile:.

Reply:
> If you think that we should also decrease the constant (currently 250) to get a faster response time, let me know smile_cat

Yes, let's go with `50` ms for all spots :)

> P.S.: I hope you can understand my not-so-clear text sweat_smile.

I think I understood everything :laughing: :+1: Thanks for the investigation!